Topics/starting-a-data-project.md

```diff
@@ -0,0 +1,254 @@
+
+
+#### Metadata and file names
+Another important issue with regard to the overall order of the project and the management of its complexity is the management of the data and metadata associated with them. 
+
+[**This awesome cheatsheet**](https://www.axiomdatascience.com/best-practices/DataManagementCheatSheet.html) contains everything you need to know about **data management and file names best practices**: keep it under your pillow!
+
+
+### Reproducibility
+
+Why are we talking about [**reproducibility**](https://en.wikipedia.org/wiki/Reproducibility)? 
+
+The field name in the Data **Science** indicates that the work process is scientific (Data Science, even with software as a component, is not pure software, which is reproducible by definition).
+
+From [this article](https://towardsdatascience.com/data-sciences-reproducibility-crisis-b87792d88513):
+> Reproducible experiments are the foundation of every scientific field and, indeed, even the scientific method itself.
+
+ Karl Popper said it best in [The Logic of Scientific Discovery](http://strangebeautiful.com/other-texts/popper-logic-scientific-discovery.pdf): “non-reproducible single occurrences are of no significance to science.” 
+
+If you’re the only person in the world who can achieve a particular result, others may find it difficult to trust you, especially if they have spent time and effort attempting to reproduce your work. 
+
+It is reckless and irresponsible to build a product or theory on a singular unconfirmed anecdote, and if you present anecdote as a reliable phenomenon, it can consume time and resources that would otherwise be spent on actual productive work.
+
+Reproducibility has a number of indirect advantages, in addition to being sure to present good results (analysis or model predictions):
+
+- It saves time in various ways, for example by saving the intermediate steps of data processing and cleaning, so that you don't have to redo all the steps
+- Allows you to automate various parts of the project workflow
+- Allows others to reproduce results
+- Allows others to understand each phase without confusion
+- Reproducible design is easier to document
+- Allows you to take over the project after months or years, and be sure to get the most out of it
+
+Here you can find articles and papers that explain to you how to ensure a high reproducibility across all the phases of the project:
+
+- [Reproducibility in Science](https://ropensci.github.io/reproducibility-guide/)
+- [Replicability is not Reproducibility: Nor is it Good Science](http://cogprints.org/7691/7/ICMLws09.pdf)
+- [Best Practices for Reproducible, Collaborative Data Science (Video)](https://www.youtube.com/watch?v=vP9Iup8xhKA)
+
+Once you've walked through the above resources, you'll be equipped with best practices to ensure that your code will be highly reproducible, and again, people will be grateful to you! 
```

Comment:
It's an interesting topic and I can recommend a [Deep Learning platform](https://valohai.com?from=3oxenia9mtr6) also see posts on [ML Catalog](https://blog.valohai.com/blog-building-a-data-catalog-for-machine-learning?from=3oxenia9mtr6) and [provenance](https://blog.valohai.com/automatic-data-provenance-for-your-ml-pipeline?from=3oxenia9mtr6) -- they focus on reproducibility from the start!

Reply:
I think its too much advanced for a beginner.. we can add them in later guides for sure.. waht do you think?

Reply:
Well if you link the best practices video why not tell them, hint in future we will cover something like this (not to worry atm) and then share the links - that way you stay at the level and still give a hint!

Reply:
Yes true, although it would be okay to give them a hint and ask them not to dwell deep into it - since you already mention the best practices video! The posts are not that advance anyways!